<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMarkers</class>
 <widget class="QDialog" name="CreateMarkers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>135</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QSlider" name="widthSlider">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="widthBox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="heightSlider">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="heightBox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
